WooCommerce/Classes/ViewRelated/Products/ProductSelector/ProductSelectorViewModel.swift

```diff
@@ -52,6 +52,15 @@ final class ProductSelectorViewModel: ObservableObject {
     ///
     @Published private(set) var productRows: [ProductRowViewModel] = []
 
+    /// Determines if multiple item selection is supported, `false` by default.
+    /// Mainly used to allow multi-selection on the Order Creation flow
+    ///
+    let supportsMultipleSelections: Bool
```

Comment:
Not a blocker, I would make it singular (`supportsMultipleSelection`). To make it more readable, I would consider using an enum for this:

```
enum SelectionStyle {
case simple
case multiple
}
```

Reply:
Thanks! Made the property singular on https://github.com/woocommerce/woocommerce-ios/pull/8935/commits/65fe4ae1998a785ccdcf879905bb5dda915171a3 . I'll leave the usage of an enum for https://github.com/woocommerce/woocommerce-ios/issues/8988 when trying to abstract properties for both Products and Coupons, I believe we may get rid of this bit entirely though, as by default all selections would be multi-selections.